### DISCUSSION TITLE
Linux binaries

### DISCUSSION THREAD

**Discussion by dca00** [navit-gps/navit#1243] [General]
Hi all,
Years ago when I used navit on a PC there used to be Linux binaries on your web site, if I remember correctly. Do you no longer provide them? Do they have to be built from source?
Many thanks!

**Comment by jkoan** [comment]
The release artefacts can be found under the releases: https://github.com/navit-gps/navit/releases

**Comment by dca00** [comment]
Great, thanks a lot! Got it working.
